xla/service/gpu/autotuning/gemm_algorithm_picker.cc

```diff
@@ -100,6 +100,8 @@ class GemmAutotuner {
   explicit GemmAutotuner(const AutotuneConfig& autotune_config)
       : autotune_config_(autotune_config) {}
 
+  const AutotuneConfig& config() const { return autotune_config_; }
+
```

Comment:
With the recent change, the changes to `gemm_algorithm_picker.cc` are not needed anymore, are they?

Reply:
yes I also did some small refactoring there: basically I moved the creation of **GemmAutotuner** object to the top function, so that it won't create a new stream for each new custom-call instruction found in the HLO module. And also made the code a bit less verbose